packages/starlight/style/markdown.css

```diff
@@ -113,3 +113,73 @@
 	border: 0;
 	border-bottom: 1px solid var(--sl-color-hairline);
 }
+
+/* <details> and <summary> styles */
+.sl-markdown-content details:not(:where(.not-content *)),
+.sl-markdown-content details[open]:hover:not(:where(.not-content *)) {
+	--sl-details-border-color: var(--sl-color-gray-4);
+
+	border-inline-start: 1px solid var(--sl-details-border-color);
+	padding-inline-start: 1rem;
+}
+.sl-markdown-content details[open]:not(:where(.not-content *)),
+.sl-markdown-content details:hover:not(:where(.not-content *)) {
+	--sl-details-border-color: var(--sl-color-gray-3);
+}
+.sl-markdown-content summary:not(:where(.not-content *)) {
+	color: var(--sl-color-white);
+	cursor: pointer;
+	display: block; /* Needed to hide the default marker in some browsers. */
+	font-weight: 600;
+	/* Expand the outline so that the marker cannot distort it. */
+	margin-inline-start: -0.5rem;
+	padding-inline-start: 0.5rem;
+}
+.sl-markdown-content details[open] > summary:not(:where(.not-content *)) {
+	margin-bottom: 1rem;
+}
+
+/* <summary> marker styles */
+.sl-markdown-content summary:not(:where(.not-content *))::marker,
+.sl-markdown-content summary:not(:where(.not-content *))::-webkit-details-marker {
+	display: none;
```

Comment:
`display: none` is invalid for summary marker, please add `list-style: none` instead